ydb/src/client_coordination/list_types.rs

```diff
@@ -100,3 +99,51 @@ impl From<RawDescribeNodeResult> for NodeDescription {
         }
     }
 }
+
+#[derive(Debug, serde::Serialize)]
+#[allow(dead_code)]
+pub struct SemaphoreSession {
```

Comment:
Need to move wrappers of grpc messages to grpc_wrapper

Reply:
fixed